yoti-sdk-api/src/main/java/com/yoti/api/client/ActivityDetails.java

```diff
@@ -25,32 +25,40 @@
     /**
      * Deprecated.  Please use getRememberMeId() instead.
      *
-     * @return user ID.
+     * @return userId (now known as rememberMeId)
      */
     @Deprecated
     String getUserId();
 
     /**
-     * Return the remember-me ID, which is a unique, stable identifier for a user in the context of an application/user
+     * Return the rememberMeId, which is a unique, stable identifier for a user in the context of an application/user
      * interaction. You can be use it to identify returning users. Note that it is different for the same user in
      * different applications.
      *
-     * @return user ID.
+     * @return rememberMeId
      */
     String getRememberMeId();
 
+    /**
+     * Return the parentRememberMeId, which is a unique, stable identifier for a user in the context of an organisation/user
```

Comment:
just `organisation` instead of `organisation/user`

Reply:
> in the context of an organisation/user interaction.

if I take out _user_, that becomes

> in the context of an organisation interaction.

I don't think that makes much sense, an interaction would normally involve two parties.

Reply:
We could probably drop `interaction` too couldn't we? So it's `a unique, stable identifier for a user in the context of an organisation.`

Reply:
Ok, changed.  So for _getRememberMeId_ we now have:
> Return the rememberMeId, which is a unique, stable identifier for a user in the context of an application.  You can use it to identify returning users. This value will be different for the same user in different applications.

And for _getParentRememberMeId_:
> Return the parentRememberMeId, which is a unique, stable identifier for a user in the context of an organisation.  You can use it to identify returning users.  This value is consistent for a given user across different apps belonging to a single organisation.